authors/apps/articles/tests/test_article_likes_and_dislikes.py

```diff
@@ -45,6 +45,22 @@ def test_cannot_like_an_article_more_than_once(self):
                                     )
         self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
 
+    def test_cannot_dislike_an_article_more_than_once(self):
```

Comment:
@meshack-mbuvi is this in the scope of your task?

Reply:
Also make sure tests are not dropping.

Reply:
I wanted to improve test coverage. All the files I interacted with are 100% covered by tests.

Reply:
I have made the requested changes.